### DISCUSSION TITLE
give me all of your zip bombs

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#856] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
please just send me as a file zip bombs of any size is permitted

**Comment by leahanderson1** [comment]
This one is about 2GB when extracted.
[lagg.zip](https://github.com/3kh0/ext-remover/files/13675208/lagg.zip)

**Reply by leahanderson1** [reply]
(make sure to extract all of the smaller zip bombs inside after extracting the main layer)

**Reply by leahanderson1** [reply]
better version:
[lagg2.zip](https://github.com/3kh0/ext-remover/files/13675464/lagg2.zip)

**Reply by rwg100** [reply, marked as answer]
thank you daddy i love you
